src/pages/starter-kit/integration/create-integration.md

```diff
@@ -93,10 +93,26 @@ To download a `.json` file containing your workspace configuration:
 
    The `<Workspace-name>.json` file downloads automatically. In this example, the file is named `977AmethystGazelle-1340225-Stage.json`. Rename the file `workspace.json` and save it in the `scripts/onboarding/config/` directory of the starter kit.  
 
-#### Create an integration in Adobe Commerce
+#### PaaS or SaaS
+
+The integration starter kit is designed to work with all Adobe Commerce versions, but only one at a time. This means that you can use either Adobe Commerce on Cloud / on Premises (PaaS) or Adobe Commerce as a Cloud Service (SaaS). By default, the starter kit will check for PaaS authentication first using **OAuth1**, before it checks for SaaS authentication using **IMS**.
+
+* For PaaS configurations, refer to [Create an integration in Adobe Commerce (PaaS only)](#create-an-integration-in-adobe-commerce-paas-only) and make sure your environment variables `COMMERCE_XXXX` are set correctly in the `.env` file.
+
+* For SaaS configurations, refer to [Create an integration in Adobe Commerce as a Cloud Service](#create-an-integration-in-adobe-commerce-as-a-cloud-service) and make sure the environment variables file does not contain `COMMERCE_XXXX` variables.
+
+<InlineAlert variant="info" slots="text"/>
+
+`app.config.yaml` has both sets of environment variables declared. These determine the runtime action context. This file works regardless of offering, so you do not need to modify it.
+
+#### Create an integration in Adobe Commerce (PaaS only)
 
 A Commerce integration generates the consumer key, consumer secret, access token, and access token secret that are required to connect to the starter kit. It also identifies the available API resources that are needed for the integration.
 
+<InlineAlert variant="info" slots="text"/>
+
+This section applies to Adobe Commerce on-premises and Adobe Commerce on cloud infrastructure (PaaS) customers only. For Adobe Commerce as a Cloud Service (SaaS) customers, see [Create an integration in Adobe Commerce as a Cloud Service](#create-an-integration-in-adobe-commerce-as-a-cloud-service).
```

Comment:
```suggestion
This section applies to **PaaS** customers only. For **SaaS** customers, see [Create an integration in Adobe Commerce as a Cloud Service](#create-an-integration-in-adobe-commerce-as-a-cloud-service).
```

It's the same topic. I don't think you need to spell out the long names again. Added bolding because you used it in the note of the subsequent section.